dango/lending/src/query.rs

```diff
@@ -41,21 +38,30 @@ pub fn query(ctx: ImmutableCtx, msg: QueryMsg) -> anyhow::Result<Json> {
     .map_err(Into::into)
 }
 
-fn query_market(ctx: ImmutableCtx, denom: Denom) -> StdResult<Market> {
-    MARKETS.load(ctx.storage, &denom)
+fn query_market(ctx: ImmutableCtx, denom: Denom) -> anyhow::Result<Market> {
+    MARKETS
+        .load(ctx.storage, &denom)?
+        .update_indices(&ctx.querier, ctx.block.timestamp)
```

Comment:
@apollo-sturdy I did this PR: https://github.com/left-curve/left-curve/pull/650, now `QuerierWrapper` is `Copy`-able, so functions should use `ctx.querier` instead of `&ctx.querier`.